src/utils/wrap_thread.cpp

```diff
@@ -94,11 +93,31 @@ wrap_thread::~wrap_thread()  {
 	}
 }
 
+std::string wrap_thread::timepoint_to_readable(const t_timepoint &tp) const {
+
+	std::time_t now_c = std::chrono::system_clock::to_time_t(tp);
+	std::stringstream ss;
+	ss << std::put_time(std::localtime(&now_c), "%Om-%Y-%T");
+
```

Comment:
1) also should be moved to some utils, maybe strings utils?
2) format "2017-04-19T12:33:19+00:00"  https://en.wikipedia.org/wiki/ISO_8601
3) timezone - it can be local time, then it will be e.g. +01:30 if it's timezone ahead 1.5 hours from UTC and so on.
4) unit test, 
a) for padding e.g. month 4 is "04" not "4"
b) and tests non-zero timezone, e.g. switch it for a moment in one test (and then restore)